tests/series/test_properties.py

```diff
@@ -49,12 +48,24 @@ def test_property_dt() -> None:
         PeriodProperties,
     )
 
+    df = DataFrame({"ts": [Timestamp(2025, 12, 6)], "td": [Timedelta(1, "s")]})
+    # python/mypy#19952: mypy gives Any
+    check(
+        assert_type(  # type: ignore[assert-type]
+            df["ts"].dt, "TimestampProperties | TimedeltaProperties | PeriodProperties"
+        ),
+        DatetimeProperties,
+    )
+    check(
+        assert_type(  # type: ignore[assert-type]
+            df["td"].dt, "TimestampProperties | TimedeltaProperties | PeriodProperties"
+        ),
+        TimedeltaProperties,
+    )
```

Comment:
can you add tests that show that one of the `TimestampProperties` works for `df["ts"]` and that one of the `TimedeltaProperties` works for `df["td"]` ?